diff -up kdeconnect-25.08.0/interfaces/conversationmessage.h.omv~ kdeconnect-25.08.0/interfaces/conversationmessage.h
--- kdeconnect-25.08.0/interfaces/conversationmessage.h.omv~	2025-08-30 02:42:04.123310976 +0200
+++ kdeconnect-25.08.0/interfaces/conversationmessage.h	2025-08-30 02:42:28.422428881 +0200
@@ -12,7 +12,38 @@
 #include "kdeconnectinterfaces_export.h"
 
 class ConversationAddress;
-class Attachment;
+
+class KDECONNECTINTERFACES_EXPORT Attachment
+{
+public:
+    Attachment()
+    {
+    }
+    Attachment(qint64 prtID, QString mimeType, QString base64EncodedFile, QString uniqueIdentifier);
+
+    qint64 partID() const
+    {
+        return m_partID;
+    }
+    QString mimeType() const
+    {
+        return m_mimeType;
+    }
+    QString base64EncodedFile() const
+    {
+        return m_base64EncodedFile;
+    }
+    QString uniqueIdentifier() const
+    {
+        return m_uniqueIdentifier;
+    }
+
+private:
+    qint64 m_partID; // Part ID of the attachment of the message
+    QString m_mimeType; // Type of attachment (image, video, audio etc.)
+    QString m_base64EncodedFile; // Base64 encoded string of a file
+    QString m_uniqueIdentifier; // unique name of the attachment
+};
 
 class KDECONNECTINTERFACES_EXPORT ConversationMessage
 {
@@ -193,38 +224,6 @@ private:
     QString m_address;
 };
 
-class KDECONNECTINTERFACES_EXPORT Attachment
-{
-public:
-    Attachment()
-    {
-    }
-    Attachment(qint64 prtID, QString mimeType, QString base64EncodedFile, QString uniqueIdentifier);
-
-    qint64 partID() const
-    {
-        return m_partID;
-    }
-    QString mimeType() const
-    {
-        return m_mimeType;
-    }
-    QString base64EncodedFile() const
-    {
-        return m_base64EncodedFile;
-    }
-    QString uniqueIdentifier() const
-    {
-        return m_uniqueIdentifier;
-    }
-
-private:
-    qint64 m_partID; // Part ID of the attachment of the message
-    QString m_mimeType; // Type of attachment (image, video, audio etc.)
-    QString m_base64EncodedFile; // Base64 encoded string of a file
-    QString m_uniqueIdentifier; // unique name of the attachment
-};
-
 inline QDBusArgument &operator<<(QDBusArgument &argument, const ConversationMessage &message)
 {
     argument.beginStructure();
