diff -urN kdeconnect-kde-0.1/kded/plugins/CMakeLists.txt kdeconnect-kde-0.1-patched/kded/plugins/CMakeLists.txt
--- kdeconnect-kde-0.1/kded/plugins/CMakeLists.txt	2013-09-06 07:40:03.000000000 +1100
+++ kdeconnect-kde-0.1-patched/kded/plugins/CMakeLists.txt	2013-09-17 20:32:09.851979926 +1100
@@ -6,7 +6,7 @@
 add_subdirectory(mpriscontrol)
 add_subdirectory(clipboard)
 add_subdirectory(telephony)
-#add_subdirectory(battery)
+add_subdirectory(battery)
 #add_subdirectory(filetransfer)
 add_subdirectory(notifications)
 
