diff -urN kdeconnect-kde-0.4.1/CMakeLists.txt kdeconnect-kde-0.4.1-patched/CMakeLists.txt
--- kdeconnect-kde-0.4.1/CMakeLists.txt	2013-10-30 11:56:11.000000000 +1100
+++ kdeconnect-kde-0.4.1-patched/CMakeLists.txt	2013-11-09 02:02:35.134650487 +1100
@@ -19,3 +19,6 @@
 add_subdirectory(plasmoid)
 
 add_subdirectory(tests)
+
+include(MacroOptionalAddSubdirectory)
+macro_optional_add_subdirectory(po)
